src/app/Component/ArchivedMessages/ArchivedMessages.js

```diff
@@ -1,9 +1,187 @@
-import React from 'react'
+import React,{useState,useEffect} from 'react'
+import "./../ChatScreen/chatscreen.css";
+import Header from "../Common/Header";
+import axios from "axios";
+import { connect } from "react-redux";
+//import { createClient } from "../../actions/actions";
+import { loaderService } from "../../../service/loaderService";
+//import CatchError from "../CatchError/CatchError";
+//import Unarchive from './../../../assests/Unarchive.svg';
+//import ArchivePinOptions from "../ChatScreen/ArchivePinOptions";
```

Comment:
Remove the commented code if not necessary

Reply:
yeah i removed